File: concept_of_programming/rect.js
class Rectangle {
  constructor(w, h) {
    this.w = w;
    this.h = h;
  }
  area() {
    return this.w * this.h;
  }
}

const r1 = new Rectangle(2, 3);
const r2 = new Rectangle(4, 5);
console.log("Area of r1: " + r1.area());
console.log("Area of r2: " + r2.area());

File: concept_of_programming/rect.cpp
#include <iostream>
using namespace std;

class Rectangle {
private:
  int w, h;

public:
  void size(int length, int height) {
    w = length;
    h = height;
  }
  int area() { return w * h; }
};
int main() {
  Rectangle r1, r2;
  r1.size(2, 3);
  r2.size(4, 5);
  cout << "Area of r1: " << r1.area() << endl;
  cout << "Area of r2: " << r2.area() << endl;
}